DateTimeCompareValidator.php

```diff
@@ -62,7 +62,11 @@ class DateTimeCompareValidator extends Validator
      * - `{compareAttribute}`: the label of the attribute to be compared with
      */
     public $message;
-
+    /**
+     *
+     * @var string with date format according to DateTime::createFromFormat specifiaction.
```

Comment:
Remove the extra newline here and change the comment to just "Date format according to DateTime::createFromFormat specification." "Specification" is spelt wrong.